RLBotPack/The Forsaken/hive.py

```diff
@@ -156,10 +156,13 @@ def debug_stack(self):
                 offset += 1
 
     def run(self):
-        if len(self.drones) == 1 and len(self.friends) == 0:
-            run_1v1(self)
-        else:
-            run_hivemind(self)
+        try:
+            if len(self.drones) == 1 and len(self.friends) == 0:
+                run_1v1(self)
+            else:
+                run_hivemind(self)
+        except:
+            pass
```

Comment:
Please consider printing the stack trace here:
```python
except:
    traceback.print_exc()
```

Then if somebody reports that your bot "stops moving sometimes" it will be possible to diagnose.

https://stackoverflow.com/questions/3702675/how-to-print-the-full-traceback-without-halting-the-program

Reply:
Good call and I changed it